plugin/fallback/fallback.go

```diff
@@ -0,0 +1,59 @@
+// Package fallback implements a fallback plugin for CoreDNS
+package fallback
+
+import (
+	"context"
+
+	"github.com/miekg/dns"
+
+	"github.com/coredns/coredns/plugin"
+	"github.com/coredns/coredns/plugin/pkg/nonwriter"
+	"github.com/coredns/coredns/plugin/proxy"
+)
+
+// Fallback plugin
```

Comment:
That's not really a comment... (also needs closing dot)

Reply:
OK. Will add to comments in fallback.go and setup.go as appropriate.